### DISCUSSION TITLE
Mapping Error for unlisted movies

### DISCUSSION THREAD

**Discussion by TechJedi51** [meisnate12/Plex-Meta-Manager#929] [Q&A]
Is there a way to have PMM recognize movies like "The Empire Strikes Back Despecialized Edition", when there is no listing in the IMDB or TMDb?

I would like to have it add custom artwork, summary etc. There seems to be an identifier.

It shows in the log as:
`Mapping Error | local://111000`

**Comment by chazlarson** [comment]
I match those as the actual movie; that will clear the error, then you could target by name.

**Reply by TechJedi51** [reply]
How do you differentiate between that and the actual movie?

**Reply by chazlarson** [reply]
In Plex?  Give them different names and posters.  Then you should be able to identify them by name in PMM.
